### PR TITLE
Add personal page for 0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01

### PR DESCRIPTION
**Pull request by solo1g**
## Description

Added personal page for 0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01 (#8).

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues
#8 

Your ENS/address: **solo1g.eth**

### CODE REVIEW

File: packages/nextjs/pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  const leftPanel = (
+    <div className="">
+      {/* eslint-disable @next/next/no-img-element */}
```
Comment (solo1g) [inline packages/nextjs/pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx:6]: Do we want [this](https://nextjs.org/docs/app/api-reference/components/image#remotepatterns) instead?

Comment (carletex) [reply packages/nextjs/pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx:6]: It's not a big deal, but using NextJS `<Image>` gives you some cool optimizations. 

https://nextjs.org/docs/app/building-your-application/optimizing/images

The problem here is that you'd need to tweak `next.config.js` to allow the URL patterns of external URLs.

**One way to solve it:** What if you copy the asset to the repo (maybe create a folder inside `public`)?

You can also leave it like this. Up to you

**Review by carletex — APPROVED** [review]
Great stuff @solo1g !!

Left a comment answering yours... but this is good for a merge. Looking good on mobile too ;)

Great job!

File: packages/nextjs/pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx (hunk 1 above)
Comment (solo1g) [reply packages/nextjs/pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx:6]: Leaving it as it is. Seems like the simplest solution. Not much point in tweaking config or hosting in public.

**Comment by miralsuthar** [comment]
packages/nextjs/pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx

I think the brackets around the address will use it as page id, rather than the personal page and will redirect every other link to this same page.

**Comment by solo1g** [comment]
> pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx

Good catch. Thanks.

**Pull request merged**